src/Microsoft.Fast.Components.FluentUI/Components/FluentDialog.razor

```diff
@@ -1,9 +1,15 @@
-<fluent-dialog Modal="@Modal" @attributes="AdditionalAttributes">@ChildContent</fluent-dialog>
+<fluent-dialog Modal="@Modal" @attributes="AdditionalAttributes" hidden="@(Hidden)">@ChildContent</fluent-dialog>
```

Comment:
Is it possible here for an author to apply the `hidden` attribute via AdditionalAttributes? If so, I'm assuming that it won't take priority, is that correct?

Reply:
Probably you could add them to the `AdditionalAttributes`, but then you would have to handle the `show` and `hide` from outside of the component? As updating the value inside the `AdditionalAttributes` parameter for the component to be hidden.

Reply:
Do we need to move `hide` to before `AdditionalAttributes` or does the order not matter here?

Reply:
Hmm actually assuming order *does* matter I think we want them ordered as you have them, otherwise the Blazor control could get out of sync with the DOM element in cases where the attribute is applied via `AdditionalAttributes`.